terraform/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "prefix"{
+    description = "The prefix used for all resources in this environment"
+}
+variable "flask_app"{
+    description = "The location of flask app"
+    sensitive   = true
+}
+variable "secret_key"{
+    description = "Flask secret ket"
+    sensitive   = true
+}
+variable "mongo_collection_name"{
+    description = "The mongo collection name"
```

Comment:
This is an example of a value that doesn't have to be marked sensitive. Not marking them as sensitive could be useful in reading the CLI output and help someone who reads the code understand which values need to be kept secret better